doc/htmldoc/installation/cmake_options.rst

```diff
@@ -38,6 +38,26 @@ Options for configuring NEST
 
 NEST allows for several configuration options for custom builds:
 
+Select built-in models
+~~~~~~~~~~~~~~~~~~~~~~
+
+By default, NEST will compile and register *all* neuron and synapse
+models that are shipped in the source distribution. This is very
+convenient for an explorative development of simulation scripts, but
+leads to quite long compilation times and is often not necessary.
+
+There are two ways to restrict the set of built-in models to tailor
+NEST to your needs:
+
++-------------------------------------+------------------------------------------------------------------------------+
+| ``-Dwith-modelset=<modelset>``      | Specify the modelset to include. Sample configurations are in the            |
```

Comment:
Could you add more documentation about the different modelsets?

Reply:
In b2ba3fd82 I have added a bit more information about what a modelset actually is and in 03282f155 I have added informative headers to the sample modelsets.

I don't want to go into more detail about the different modelsets themselves in the documentation, as they might change over time and keeping the documentation consistent with the actual modelsets would be hard.